### PR TITLE
openyurt fuzzer feature

### PR DESCRIPTION
**Pull request by huiwq1990**
Signed-off-by: huiwq1990 <huiwq1990@163.com>

/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### Which issue(s) this PR fixes:

Fixes https://github.com/openyurtio/openyurt/issues/900

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@huiwq1990 I want to invite you to introduce fuzz test to everybody of OpenYurt community. how about make a presentation of fuzz test in the community meeting?

File: pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+
```
Comment (rambohe-ch) [inline pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go:1]: please add license header.

File: pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+)
```
Comment (rambohe-ch) [inline pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go:20]: import sequences:
1. golang package
2. k8s or github package
3. openyurtio package

File: pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go (hunk 1 above)
Comment (huiwq1990) [reply pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go]: fixed

File: pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go (hunk 2 above)
Comment (huiwq1990) [reply pkg/yurtappmanager/controller/yurtappdaemon/yurtappdaemon_fuzzer.go]: fixed

**Comment by huiwq1990** [comment]
@rambohe-ch https://github.com/cncf/cncf-fuzzing/pull/208 is merged, https://github.com/google/oss-fuzz/pull/8062 is waiting for this pr ready.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**